src/App.js

```diff
@@ -3,27 +3,36 @@ import { Routes, Route, useNavigate } from "react-router-dom";
 
 import { useVideosDataProvider } from "./Context/VideosDataContext/VideosDataProvider";
 import { useStateContext } from "./Context/StateContext/StateProvider";
+import { useAuthProvider } from "./Context/AuthContext/AuthProvider";
 import {
   Home,
   Navbar,
   VideoDetailPage,
   Login,
   SignUp,
+  LikedVideo,
   MyProfile,
   Setting,
 } from "./Components";
-import { getVideos, getLikedVideos } from "./utilities/backendRequest";
+import {
+  getVideos,
+  getUserProfile,
+  getLikedVideos,
+} from "./utilities/backendRequest";
 
 import "./styles.css";
 
 export const App = () => {
   const navigate = useNavigate();
   const { setVideosList } = useVideosDataProvider();
   const { state, dispatch } = useStateContext();
+  const { setUserProfile } = useAuthProvider();
   const token = localStorage.getItem("token");
 
   useEffect(() => {
     getVideos(setVideosList);
+    getLikedVideos(dispatch, token);
+    getUserProfile(setUserProfile, token);
     if (token) {
       getLikedVideos(state, dispatch);
```

Comment:
Suggestion: while naming a function you should use the `get` suffix only if it's returning something.
But here these functions don't seem to be pure, so you may name it something else.